.vscode/settings.json

```diff
@@ -1,13 +1,23 @@
 {
-  "editor.defaultFormatter": "rvest.vs-code-prettier-eslint",
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Did you notice any differences after switching this?

Reply:
Yeah not having to have that extra so I was able to uninstall it and keep prettier&eslint exts